apiclient/exceptions.py

```diff
@@ -59,3 +66,24 @@ class UnexpectedError(APIRequestError):
     """An unexpected error occurred when making the request."""
 
     pass
+
+
+class BaseExceptionHandler:
+    @staticmethod
+    def get_exception_class(response: Response) -> Type[APIRequestError]:
+        raise NotImplementedError
+
+
+class ExceptionHandler(BaseExceptionHandler):
```

Comment:
small request: could you rename this to `ErrorHandler`  as it's not actually handling an exception, but rather handling an error to return an exception type.  I think this should also be moved to a different module `error_handlers.py`, to keep the `exceptions.py` simply for defining exception types.

Reply:
👌